cellprofiler/modules/graphpartition.py

```diff
@@ -0,0 +1,99 @@
+"""
+
+Graph partition
+
+"""
+
+import cellprofiler.image
+import cellprofiler.module
+import cellprofiler.object
+import cellprofiler.setting
+import skimage.measure
+import skimage.segmentation
+
+
+class GraphPartition(cellprofiler.module.Module):
+    category = "Image segmentation"
+    module_name = "Graph partition"
+    variable_revision_number = 1
+
+    def create_settings(self):
+        self.image_name = cellprofiler.setting.ImageNameSubscriber(
+            "Image"
+        )
+
+        self.object_name = cellprofiler.setting.ObjectNameProvider(
+            "Object",
+            ""
+        )
+
+        self.markers_name = cellprofiler.setting.ImageNameSubscriber(
+            "Markers"
+        )
+
+    def settings(self):
+        return [
+            self.image_name,
+            self.object_name,
+            self.markers_name
+        ]
+
+    def visible_settings(self):
+        return [
+            self.image_name,
+            self.object_name,
+            self.markers_name
+        ]
+
+    def run(self, workspace):
+        image_name = self.image_name.value
+
+        object_name = self.object_name.value
+
+        images = workspace.image_set
+
+        image = images.get_image(image_name)
+
+        dimensions = image.dimensions
+
+        image_data = image.pixel_data
+
+        markers_name = self.markers_name.value
+
+        markers = images.get_image(markers_name)
+
+        markers_data = markers.pixel_data
+
+        label_data = skimage.segmentation.random_walker(
+            data=image_data,
+            labels=markers_data,
+            beta=130,
+            mode="bf",
+            tol=0.001,
+            copy=True,
+            multichannel=False,
+            return_full_prob=False,
+            spacing=None
+        )
+
+        label_data = skimage.measure.label(label_data)
```

Comment:
`skimage.measure.label` returns `labels` and `num`. Only `labels` are needed here.